cockatrice/src/remoteclient.cpp

```diff
@@ -135,6 +136,13 @@ void RemoteClient::processConnectionClosedEvent(const Event_ConnectionClosed & /
 void RemoteClient::loginResponse(const Response &response)
 {
     const Response_Login &resp = response.GetExtension(Response_Login::ext);
+    
+    QString possibleMissingFeatures;
```

Comment:
I actually started out by using a QSet<QString> but ran into an issue with doing so.  At this point though I don't recall what the issue was and it very well could have been something to do with me taking a different approach and then changing it after the failures occurred.